### DISCUSSION TITLE
Getting the correct units for a body force, where a body forces acts on a single voxel (not an entire surface/edge)

### DISCUSSION THREAD

**Discussion by kurtenkera** [idaholab/moose#27926] [Q&A Modules: Solid mechanics]
Hi, I am attempting to solve the PDEs shown below:

![image](https://github.com/idaholab/moose/assets/127472392/87cb42db-e309-4dee-b3d8-a08ce6f36298)

I am trying to model the cube seen below, and I have four different forces $\bf{F}_1$, $\bf{F}_2$, $\bf{F}_3$ and $\bf{F}_4$ that I am modelling as body forces. 

**My question:** I know that body forces must have units of **force/volume**. In my case, the forces aren't acting on a 'volume', but rather on single voxels (i.e., they are localised to the corners of the box in this case). What volume must I divide these forces by: (a) the volume of the entire cube being modelled? Or (b) the volume of a single voxel on which these localized forces are acting on?

![image](https://github.com/idaholab/moose/assets/127472392/6569bb2f-45a2-46cb-ac16-1ed8cd3f0440)

**Comment by GiudGiud** [comment]
thanks for detailed post.
See https://github.com/idaholab/moose/discussions/27714#discussioncomment-9808140
